WordPress/gutenberg
# Search inputs modal box width issue fix

## What?
Fixes https://github.com/WordPress/gutenberg/issues/65495

## Why?
Search box width in some modal dialogs are a little too wide and this PR fixes it.

## How?
Fixes the search box width in some modal dialogs to optimum size.

## Testing Instructions

1. Go the Site editor.
2. Edit a template.
3. Add a template part.
4. In the template part placeholder, click 'Choose'.
5. The 'Choose a template part' modal dialog opens full width.
6. Observe the search input inside the modal dialog is now in correct size.

### Testing Instructions for Keyboard


## Screenshots or screencast 



**Before**

 ![search_box_width_before](https://github.com/user-attachments/assets/ec0edea8-8d5f-40ed-997f-0952a6b16a41)
 
 **After**

![search_box_width_after](https://github.com/user-attachments/assets/0df840fd-bffc-47b0-97cf-c0cc24e5ae2a)

**Video:**

https://github.com/user-attachments/assets/4ae3fc1d-1b2f-4bcc-a01e-90fda358f404
